### DISCUSSION TITLE
CPU temp limit not working anymore?

### DISCUSSION THREAD

**Discussion by Epyon83** [seerge/g-helper#2421] [General]
I used in the past 85° max temp for CPU
Currently running 0.1640 and after somehow everything got reset after the update the limit doesnt seem to work.
I am right now at 93° and it wont trottle down. Anyone else got this issue?
ROG Strix G713RW
![image](https://github.com/seerge/g-helper/assets/30730011/93c5fa45-0624-4f54-8049-ff65fcf2795f)

**Comment by seerge** [comment]
@Epyon83 you need to check ``Auto Apply,`` (Automatisch anwenden) or AMD drivers would keep resetting it.

Also keep in mind, that temp limit, is just CPU Throttle temperature (i.e. your CPU will just drop power consumption if it reaches it). 

But there is no magic here and if you have bad liquid metal / thermal paste / slow fans / or just get extra heat from GPU it can still go over that temp.
